pr_description_toast.md

```
@@ -0,0 +1,21 @@
+# Feature: Replace Default Alerts with Toast Notifications
+
+## Description
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)